test/integration/client-side-encryption/client_side_encryption.spec.test.ts

```diff
@@ -97,6 +97,9 @@ describe('Client Side Encryption (Legacy)', function () {
     }
 
     if (isServerless) {
+      if (description === "Compact works with 'range' fields") {
+        return 'TODO(CLOUDP-267864): fix compactStructuredEncryptionData against serverless';
```

Comment:
Can we file a NODE project ticket that's blocked on the CLOUDP ticket and link that instead. I feel we're more likely to action it faster if we have the NODE ticket.

Reply:
Done.